epochX/cudacpp/CODEGEN/PLUGIN/CUDACPP_SA_OUTPUT/launch_plugin.py

```diff
@@ -33,10 +33,14 @@ def compile(self, *args, **opts):
         if 'cwd' in opts and os.path.basename(opts['cwd']) == 'Source':
             path = pjoin(opts['cwd'], 'make_opts')
             common_run_interface.CommonRunCmd.update_make_opts_full(path,
-                {'FPTYPE': self.run_card['floating_type'] })
+                {'FPTYPE': self.run_card['cudacpp_fptype'] })
             misc.sprint('FPTYPE checked')
         cudacpp_supported_backends = [ 'fortran', 'cuda', 'hip', 'cpp', 'cppnone', 'cppsse4', 'cppavx2', 'cpp512y', 'cpp512z', 'cppauto' ]
         if args and args[0][0] == 'madevent' and hasattr(self, 'run_card'):            
+            if self.run_card['cudacpp_bldall'] == 'True': # pre-build all backends #945
```

Comment:
This sounds bad why cudacpp_bldall is not a boolean?

Reply:
Hi @oliviermattelaer thanks. I make these changes in PR #948 which is smaller and more manageable, so hopefully can be reviewed/approved more easily. I changed to boolean in https://github.com/madgraph5/madgraph4gpu/pull/948/commits/c68a9a7dcd282b1f7114e5c5db5cf937fd47228f